lib/spack/external/__init__.py

```diff
@@ -17,7 +17,7 @@
 --------
 
 * Homepage: https://pypi.python.org/pypi/argparse
-* Usage: We include our own version to be Python 2.6 compatible.
+* Usage: We include our own version to be Python 3.X compatible.
```

Comment:
Should this be 2.X compatible? I'm surprised we still need to vendor this even in Python 2.7.

Reply:
2.7 unfortunately doesn't have aliases.

Reply:
@adamjstewart See for instance:

https://github.com/spack/spack/blob/feb66cba011446da25b90982de7a29d869d9c53e/lib/spack/spack/cmd/compiler.py#L35-L37